Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/MainActivity.java

```diff
@@ -230,13 +234,31 @@ private void parseServer() {
     }
   }
 
-  // Converts time in String format hh:mm:ss.MsMsMs (Eg: 08:47:56.637) to Integer milliseconds
-  private Integer convertStringTimeToInt(String currentTime) {
+  // Converts time in String format hh:mm:ss.MsMsMs (Eg: 08:47:56.637) to Integer millisecondsy
+  private Integer convertStringTimeToInt(@NotNull String currentTime) {
     Integer result = 0;
     result += (Integer.parseInt(currentTime.substring(0, 2))) * 3600000;
     result += (Integer.parseInt(currentTime.substring(3, 5))) * 60000;
     result += (Integer.parseInt(currentTime.substring(6, 8))) * 1000;
     result += (Integer.parseInt(currentTime.substring(9, 12)));
     return result;
   }
+
+  private void showResults() {
+    if (deltaServer.size() < 2 || deltaOCR.size() < 2) {
+      Log.d(tag, "No results to show");
+      return;
+    }
+    Log.d("Show Results", "Video start timestamp:" + CameraFragment.Companion.getRecordingStartTime());
+    String serverChar = "";
+    Integer sync_offset = deltaOCR.get(0) - convertStringTimeToInt(CameraFragment.Companion.getRecordingStartTime());
```

Comment:
`deltaOCR[0]` have the starting timestamp when tests are begun on laptop. After 2 seconds of that, the keys are smulated.